schema/post.ts

```diff
@@ -25,6 +25,7 @@ export const SavePostSchema = z.object({
   canonicalUrl: z.optional(z.string().trim().url()),
   tags: z.string().array().max(5).optional(),
   published: z.string().datetime().optional(),
+  seriesName: z.string().trim().optional()
```

Comment:
[savePostSchema reference](https://docs.google.com/document/d/1xhTqymwo24xhlVUiUlgMdZ3ZysAvzu7stKqwVyySGsA/edit?usp=sharing)

Previously review comments were raised stating that seriesName would cause issues while joining table as it will not be unique. 

However, the savePostSchema is not the schema for the post table. The post table itself has seriesId as a foreign key. Please refer to the link above, where I have included screenshots of the post table schema and how the savePostSchema is currently being used.